modules/swagger-annotations/src/main/java/io/swagger/oas/annotations/security/OAuthFlow.java

```diff
@@ -23,34 +23,30 @@
 
 
 /**
- * 
- *
- * 
+ * Configuration details for a supported OAuth Flow.
  **/
-
-
 @Target({  })
 @Retention(RetentionPolicy.RUNTIME)
 @Inherited
 public @interface OAuthFlow {
   /**
-   * 
+   * Required. The authorization URL to be used for this flow. This must be in the form of a URL.  Applies to oauth2 ("implicit", "authorizationCode") type.
    **/
   String authorizationUrl() default "";
 
   /**
-   * 
+   * Required. The token URL to be used for this flow. This must be in the form of a URL.  Applies to oauth2 ("password", "clientCredentials", "authorizationCode") type.
    **/
   String tokenUrl() default "";
 
   /**
-   * 
+   * The URL to be used for obtaining refresh tokens. This must be in the form of a URL.  Applies to oauth2 type.
    **/
   String refreshUrl() default "";
 
   /**
-   * 
+   * Required. The available scopes for the OAuth2 security scheme.  Applies to oauth2 type.
    **/
-  Scopes scopes() default @Scopes;
+  OAuthScope[] scopes() default {};
```

Comment:
Is this just a naming change plus change in default?  If so, I get it, but this deviates from the naming in the spec, which will create confusion